src/app/diagnostics/views/DiagnosticsView.vue

```diff
@@ -1,58 +1,65 @@
 <template>
-  <RouteView>
-    <RouteTitle
-      :title="t('diagnostics.routes.item.title')"
-    />
-    <AppView
-      :breadcrumbs="[
-        {
-          to: {
-            name: 'diagnostics',
-          },
-          text: t('diagnostics.routes.item.breadcrumbs')
-        },
-      ]"
+  <RouteView name="diagnostics">
+    <DataSource
+      v-slot="{ data, error }: ConfigSource"
+      :src="`/config`"
     >
-      <KCard>
-        <template #body>
-          <LoadingBlock v-if="code === null" />
-
-          <CodeBlock
-            v-else
-            id="code-block-diagnostics"
-            language="json"
-            :code="code"
-            is-searchable
-            query-key="diagnostics"
-          />
+      <AppView
+        :breadcrumbs="[
+          {
+            to: {
+              name: 'diagnostics',
+            },
+            text: t('diagnostics.routes.item.breadcrumbs')
+          },
+        ]"
+      >
+        <template #title>
+          <h1>
+            <RouteTitle
+              :title="t('diagnostics.routes.item.title')"
+              :render="true"
+            />
+          </h1>
         </template>
-      </KCard>
-    </AppView>
+
+        <KCard>
+          <template #body>
+            <LoadingBlock v-if="data === undefined" />
+
+            <ErrorBlock
+              v-else-if="error"
+              :error="error"
+            />
+
+            <CodeBlock
+              v-else
+              id="code-block-diagnostics"
+              data-testid="code-block-diagnostics"
```

Comment:
just out of curiocity why do we need this with the same value as `id`?

Reply:
It could be a different value. I added the `data-testid` with that value because it doesn’t _need_ to be different and because I don’t want to use `id`s in tests if I can avoid it. This way, tests don’t depend on IDs and classes being the same.